index.js

```diff
@@ -5,14 +5,41 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "teal"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const img = document.querySelector(".traveler img")
+const h2 = document.querySelector(".traveler h2")
+const em = document.querySelector(".traveler em")
```

Comment:
Your querySelectors here isn't quite right -- since the div has an `id` of profile:

```html
<div id="profile">
```

You'd need to use the id selector (`#`) instead of class (`.`):

```js
const img = document.querySelector("#traveler img")
const h2 = document.querySelector("#traveler h2")
const em = document.querySelector("#traveler em")
```